app/javascript/schools/shared/shared.css

```diff
@@ -312,7 +312,7 @@
 }
 
 .toggle-button__button--active {
-  @apply bg-primary-100 text-primary-600;
+  @apply bg-primary-100 text-primary-600 border-primary-500 z-10;
```

Comment:
<img width="839" alt="Screenshot 2021-12-22 at 1 11 08 PM" src="https://user-images.githubusercontent.com/16238306/147107689-666af046-00e3-4ada-b78f-94a322032db0.png">

@irajsuhail The active toggle button is overlapping on the footer section.

Reply:
@vinutv I updated styles to make them more resilient in different use-cases.